base/cvd/cuttlefish/host/commands/cvd/cli/request_context.cpp

```diff
@@ -104,7 +101,7 @@ RequestContext::RequestContext(
 
 Result<CvdCommandHandler*> RequestContext::Handler(
     const CommandRequest& request) {
-  return RequestHandler(request, request_handlers_);
+  return CF_EXPECT(RequestHandler(request, request_handlers_));
```

Comment:
The `CF_EXPECT` here is load-bearing, it includes `RequestContext::Handler` in the output stack trace.

Reply:
Sorry I got the old / new versions mixed up.